FSJ_django20_project/FSJ/forms_adjudicator.py

```diff
@@ -0,0 +1,24 @@
+from .models_adjudicator import Adjudicator
+from django.forms import ModelForm
+
+class AdjudicatorForm(ModelForm):
+
+    class Meta:
+        model = Adjudicator
+        fields = ('ccid', 'first_name', 'last_name', 'email', 'lang_pref', 'usesrid')
+        
+    def __init__(self, *args, **kwargs):
+        super(AdjudicatorForm, self).__init__(*args, **kwargs)               
+        
+    
+class AdjudicatorRestrictedForm(ModelForm):
+
+    class Meta:
+        model = Adjudicator
+        fields = ('ccid', 'first_name', 'last_name', 'email', 'lang_pref')
```

Comment:
userid not shown on restricted profiles? that's the employee id / student id field (though we will need to change the label for the various cases at some point)

Reply:
Is that something students see on their own profile? I can put it back on